test/unit/teacher_set_test.rb

```diff
@@ -220,7 +223,7 @@ class TeacherSetTest < ActiveSupport::TestCase
           resp = TeacherSet.create_or_update_teacher_set(SIERRA_USER["data"][0])
         end
       end
-      assert_equal(@teacher_set6.bnumber, resp.bnumber)
+      assert_equal("b#{bib_id}", resp.bnumber)
```

Comment:
@thisisstephenbetts  this is existing test case was failing some times. I am trying to fix those test case. We don't have any proper test cases for rake methods.